src/inmanta/logging.py

```diff
@@ -443,7 +443,11 @@ def get_logging_config_from_options(
                     "level": "DEBUG",
                     "propagate": True,
                     "handlers": ["core_tornado_debug_log_handler"],
-                }
+                },
+                "sqlalchemy.engine": {
+                    "level": "INFO",
```

Comment:
De we want this to be fixed to info or do we want it to go up and down with the server?

Reply:
Does it make a difference?
`sqlalchemy.engine`  will emit the log at `INFO`
if the  server is at `WARNING` it will not show the log, right?

Reply:
I guess it doesn't hurt and might be useful with other possible sqlalchemy logs at different levels